bin/varnishd/cache/cache_backend.c

```diff
@@ -208,6 +208,18 @@ vbe_dir_getfd(VRT_CTX, struct worker *wrk, VCL_BACKEND dir, struct backend *bp,
 		VSC_C_main->backend_unhealthy++;
 		return (NULL);
 	}
+
+	AZ(bo->htc);
+	bo->htc = WS_Alloc(bo->ws, sizeof *bo->htc);
+	/* XXX: we may want to detect the ws overflow sooner */
```

Comment:
The XXX is resolved by the move.
FTR, I had pointed this out here: https://github.com/varnishcache/varnish-cache/pull/4030#discussion_r1494803500

Reply:
This has been removed from this PR, I will push a separate PR for the other commits.